### DISCUSSION TITLE
Issues with academic affiliation.

### DISCUSSION THREAD

**Discussion by panda14lama** [community/community#144521] [GitHub Education]
### Hi everyone,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Nov 7, 2024 for [@panda14lama](https://github.com/panda14lama).
i have send in countless students forms but none off them works. what do i do?????

**Comment by versenilvis** [comment]
1. Check the Requirements
Ensure that you meet the eligibility requirements for the Student Developer Pack or academic verification (e.g., currently enrolled in an accredited university or school).
Make sure you are submitting the right type of student document (e.g., student ID, enrollment letter, or transcript) in a clear, readable format.
2. Make Sure the Documents Are Correct and Clear
The documents you submit must clearly display:
Your name
The name of the institution
Your enrollment status or date
Ensure the file is in a supported format (PDF, JPG, PNG) and readable, without any blurriness or cutoff details.
3. Ensure the Document Is Recent
GitHub may reject older documents that don’t show current enrollment. Try submitting a recent document, such as a current enrollment letter or a transcript with recent academic terms.
4. Check Your Email and Account
Review your email inbox and spam folder for any additional information or feedback from GitHub regarding the rejection.
Make sure your GitHub account is fully updated with correct information (like your school name and email).
5. Try Using Your School Email Address
If you haven’t already, try registering or verifying using your .edu school email address. GitHub often automatically verifies students using educational email addresses.
6. Contact GitHub Support
If you have already submitted multiple attempts and none of them were successful, you should reach out directly to GitHub’s support team for more specific assistance. Provide them with the details of your situation and ask for guidance.
GitHub Support: https://support.github.com/contact
7. Explore Other Verification Options
If you’re still unable to get verified via the usual method, some schools have specific portals or partners (like Credly or other services) for verifying student status. If your school uses one of these, try submitting through those channels.
8. Check for Known Issues
Sometimes, there may be temporary issues or delays with GitHub’s verification process. You can search for any ongoing issues on forums, or check GitHub’s status page for updates.
